stubs/pywin32/win32/win32clipboard.pyi

```diff
@@ -1,11 +1,13 @@
+from typing import Any, Final
+
 from win32.lib.pywintypes import error as error
 
 def ChangeClipboardChain(hWndRemove: int, hWndNewNext: int, /): ...
 def CloseClipboard(): ...
 def CountClipboardFormats(): ...
 def EmptyClipboard(): ...
 def EnumClipboardFormats(_format: int = ..., /): ...
-def GetClipboardData(_format, /) -> str: ...
+def GetClipboardData(_format, /) -> Any: ...
```

Comment:
We now require an explanation (comment) of the possible types allowed for `Any` types.